### PR TITLE
[DEV-1252] storybook setup in the same directory

### PR DESCRIPTION
**Pull request by tommaso1**
#### List of Changes

Add storybook to test nextjs-wbsite UI component

#### Motivation and Context

Test nextjs-wbsite UI component

#### How Has This Been Tested?

Running this command: `npm run storybook -w nextjs-website`  and navigate on: `http://localhost:6006/`

#### Screenshots (if appropriate):
image

#### Types of changes

- [x] Chore (nothing changes by a user perspective)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by MarcoPonchia — APPROVED** [review]

File: apps/nextjs-website/.storybook/main.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import type { StorybookConfig } from "@storybook/nextjs";
+
+import { join, dirname } from "path";
+
+/**
+ * This function is used to resolve the absolute path of a package.
+ * It is needed in projects that use Yarn PnP or are set up within a monorepo.
+ */
+function getAbsolutePath(value: string): any {
```
Comment (kin0992) [inline apps/nextjs-website/.storybook/main.ts:9]:
```suggestion
function getAbsolutePath(value: string) {
```

File: apps/nextjs-website/.storybook/main.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import type { StorybookConfig } from "@storybook/nextjs";
+
+import { join, dirname } from "path";
+
+/**
+ * This function is used to resolve the absolute path of a package.
+ * It is needed in projects that use Yarn PnP or are set up within a monorepo.
```
Comment (kin0992) [inline apps/nextjs-website/.storybook/main.ts:7]: According to this comment, we don't need this function, do we?
We are not using `Yarn`

File: package.json [hunk 3]
```diff
@@ -11,6 +11,7 @@
     "dev": "turbo run dev",
     "lint": "turbo run lint",
     "test": "turbo run test",
+    "storybook": "cd ./apps/storybook; npm run storybook",
```
Comment (kin0992) [inline package.json:14]: We should run the command using `turbo`

File: README.md [hunk 4]
```diff
@@ -35,6 +35,16 @@ Open [http://localhost:3000](http://localhost:3000) with your browser to see the
 
 Open [http://localhost:1337/admin/](http://localhost:1337/admin/) with your browser to see the CMS website.
 
+### Run the storybook for nextjs-website locally
+
+Run the following command from the root folder.
+
+``` bash
+npm run storybook -w nextjs-website
```
Comment (kin0992) [inline README.md:43]: Running this command, the storybook automatically opens in the browser. Can we disable this functionality?

**Review by kin0992 — CHANGES_REQUESTED** [review]
Also, check the CI: there are some errors

File: README.md [hunk 5]
```diff
@@ -35,6 +35,16 @@ Open [http://localhost:3000](http://localhost:3000) with your browser to see the
 
 Open [http://localhost:1337/admin/](http://localhost:1337/admin/) with your browser to see the CMS website.
 
+### Run the storybook for nextjs-website locally
+
+Run the following command from the root folder.
+
+``` bash
+npm run storybook --no-open -w nextjs-website
```
Comment (kin0992) [inline README.md:43]:
```suggestion
npm run storybook -w nextjs-website
```
why not putting directly in the script within the `package.json`?

**Review by kin0992 — CHANGES_REQUESTED** [review]
Still the CI is failing

**Comment by kin0992** [comment]
And BTW, why should we keep storybook within the `nextjs-website` app and not creating a separate application (e.g. `apps/storybook`?

**Comment by marcobottaro** [comment]
Converted to draft while looking for an alternative configuration by following previous installation set up

**Pull request closed without merging**